src/20_Components/amp-sidebar.html

```diff
@@ -91,5 +101,110 @@
+      Please resize the browser window to view the responsiveness for amp-sidebar's toolbar.
+    </p>
+
+  <!-- ## Toolbar -->
+  <!--
+    `toolbar` elements have their own requirments:
+
+    The sidebar may implement toolbars by adding nav elements with the `toolbar` attribute and `toolbar-target` attribute.
+
+    The nav element must be a child of `<amp-sidebar>` and follow this format: `<nav toolbar="(media-query)" toolbar-target="elementID">`.
+
+    The nav containing the toolbar attribute must only contain a single `<ul>` element, that contains `<li>` elements.
+
+    The `<li>` elements may contain any valid HTML elements (supported by AMP), or any of the AMP elements that `<amp-sidebar>` supports.
+
+    Toolbar behavior is only applied while the `toolbar` attribute media-query is valid. Also, an element with the `toolbar-target` attribute id must exist on the page for the toolbar to be applied.
+  -->
+    <div>
+      <amp-sidebar id="sidebar-left" layout="nodisplay" side="left">
+        <amp-img class="amp-close-image" src="/img/ic_close_black_18dp_2x.png" width="20" height="20" alt="close sidebar" on="tap:sidebar-left.close" role="button" tabindex="0"></amp-img>
+        <nav toolbar="(max-width: 56rem)" toolbar-target="target-element-left">
+          <ul>
+            <li><a href="/">Home</a></li>
+            <li> Nav item 1</li>
+          </ul>
+        </nav>
+        <ul>
+          <li>
+            <amp-fit-text width="220" height="20" layout="responsive" max-font-size="24">
+              Nav item 2 - &lt;amp-fit-text&gt;
+            </amp-fit-text>
+          </li>
+          <li>
+            <amp-fit-text width="220" height="20" layout="responsive" max-font-size="24">
+              Nav item 3 - &lt;amp-fit-text&gt; longer text
+            </amp-fit-text>
+          </li>
+          <li> Nav item 4 - Image<amp-img class="amp-sidebar-image" src="/img/favicon.png" width="20" height="20" alt="an image"></amp-img></li>
+          <li> Nav item 5</li>
+          <li> Nav item 6</li>
+        </ul>
+      </amp-sidebar>
+
+
+      <button on="tap:sidebar-left.toggle" class="ampstart-btn caps m2">Toggle sidebar</button>
+
+
+      <div id="target-element-left">
+      </div>
+    </div>
+
+  <!-- ## Styling Toolbar -->
+  <!--
+    The `toolbar` element within the `<amp-sidebar>` element, will have classes applied to the element depending if the `toolbar-target` element is shown or hidden.
+
+    This is useful for applying different styles on the `toolbar` element and then `toolbar-target` element.
+
+    The classes are `amp-sidebar-toolbar-target-shown`, and `amp-sidebar-toolbar-target-hidden`.
+
+    The class `amp-sidebar-toolbar-target-shown` is applied to the `toolbar` element when the `toolbar-target` element is shown.
+
+    The class `amp-sidebar-toolbar-target-hidden` is applied to the `toolbar` element when the `toolbar-target` element is hidden.
+  -->
+    <div>
+      <style amp-custom="">
```

Comment:
This will probably make the sample invalid AMP. Instead embed the snippet into the doc via:

```
```css
<style amp-custom="">
...
</style>
```
```